// @expo-google-fonts/noto-sans-arabic/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const NotoSansArabic_100Thin = require('./100Thin/NotoSansArabic_100Thin.ttf');
export const NotoSansArabic_200ExtraLight = require('./200ExtraLight/NotoSansArabic_200ExtraLight.ttf');
export const NotoSansArabic_300Light = require('./300Light/NotoSansArabic_300Light.ttf');
export const NotoSansArabic_400Regular = require('./400Regular/NotoSansArabic_400Regular.ttf');
export const NotoSansArabic_500Medium = require('./500Medium/NotoSansArabic_500Medium.ttf');
export const NotoSansArabic_600SemiBold = require('./600SemiBold/NotoSansArabic_600SemiBold.ttf');
export const NotoSansArabic_700Bold = require('./700Bold/NotoSansArabic_700Bold.ttf');
export const NotoSansArabic_800ExtraBold = require('./800ExtraBold/NotoSansArabic_800ExtraBold.ttf');
export const NotoSansArabic_900Black = require('./900Black/NotoSansArabic_900Black.ttf');
